src/components/modebar/buttons.js

```diff
@@ -47,19 +47,26 @@ var modeBarButtons = module.exports = {};
 
 modeBarButtons.toImage = {
     name: 'toImage',
-    title: function(gd) { return _(gd, 'Download plot as a png'); },
+    title: function(gd) { return _(gd, 'Download plot'); },
     icon: Icons.camera,
     click: function(gd) {
-        var format = 'png';
+        var toImageButtonDefaults = gd._context.toImageButtonDefaults;
+        var opts = {format: toImageButtonDefaults.format || 'png'};
 
         Lib.notifier(_(gd, 'Taking snapshot - this may take a few seconds'), 'long');
 
         if(Lib.isIE()) {
             Lib.notifier(_(gd, 'IE only supports svg.  Changing format to svg.'), 'long');
-            format = 'svg';
+            opts.format = 'svg';
```

Comment:
also we should change the conditional to `if(Lib.isIE() && opts.format !== 'svg')`